lib/platform/github/index.ts

```diff
@@ -109,6 +109,8 @@ export async function initPlatform({
     throw new Error('Init: You must configure a GitHub personal access token');
   }
 
+  platformConfig.isGHApp = token.startsWith('x-access-token:');
```

Comment:
@rarkins We should use this in `initRepo` in a separate pr, so the code there will be simplified.

Reply:
OK. I need to double check that these changes don't break the app or WSOP too